polymetis/examples/franka_remote.py

```diff
@@ -0,0 +1,113 @@
+import Pyro4
+from polymetis import RobotInterface
+import torch
+
+Pyro4.config.SERIALIZERS_ACCEPTED.add("pickle")
+Pyro4.config.ITER_STREAMING = True
+Pyro4.config.PICKLE_PROTOCOL_VERSION = 4
+
+
+@Pyro4.expose
+class RemoteFranka(object):
+    def __init__(self):
+        self.robot = RobotInterface(ip_address="localhost")
+
+    def connect(self):
+        if self.robot == None:
+            self.robot = RobotInterface(ip_address="localhost")
+
+    def go_home(self):
+        # Reset
+        self.robot.go_home()
+        joint_pos = self.robot.get_joint_angles()
+
+    def move(self, joint, vel):
```

Comment:
To be more clear, the `joint` argument is actually `joint_index`, and `vel` is actually subtracted from `time_to_go`, right? I think the argument names should be  more clear.

Also, in this situation you can only increase each joint's angles, not decrease then.

Reply:
I will select clearer naming for the arguments.